README.md

```diff
@@ -197,3 +173,27 @@ docker_credentials = {
   username = var.dockerhub_username
   password = var.dockerhub_password
 }
+```
+
+## Deployment
+
+The deployment of the `prometheus_all` module is managed in the [deployment](deployment) directory. This includes all internal and external metric target configuration.
+
+### Applying Terraform
+
+Terraform is applied to the `monitoring` space of the GOV.UK Notify Org - you must have admin access to this space. 
+
+The Terraform state stored in S3 in the `notify-tools` AWS account - you must have access to this account and write access to the state bucket.
+
+From the `/deployment` dir:
+
+```
+terraform init
+gds aws notify-tools-admin -- terraform apply
+```
+
+You will be prompted to enter your GOV.UK PaaS authentication credentials or and SSO passcode if SSO is enabled.
+
+Obtain an SSO passcode by signing into GOV.UK PaaS and navigating to: [login.cloud.service.gov.uk/passcode](https://login.cloud.service.gov.uk/passcode)
+
+Copy the passcode and paste into the terminal prompt.
```

Comment:
It's worth pointing out that this is something that they'll need to do only once per day (I think PaaS tokens have a TTL of 8h?)

Reply:
Mine asks every time for some reason